pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -960,6 +965,18 @@ type OIDCProviderConfiguration struct {
 	SkipTLSVerify *bool `json:"skipTLSVerify,omitempty"`
 }
 
+type KubeLBSettings struct {
+	// Kubeconfig is reference to the Kubeconfig for the kubeLB management cluster.
+	Kubeconfig corev1.ObjectReference `json:"kubeconfig"`
+}
+
+type KubeLBDatacenterSettings struct {
+	// Used to configure and override the default kubeLB settings.
+	KubeLBSettings `json:",inline"`
+	// Enabled is used to enable/disable kubeLB for the datacenter. This is used to control whether installing kubelb is allowed or not for the datacenter.
```

Comment:
Please consistently use "KubeLB", not "kubelb".

Reply:
Fixed